packages/xo-web/src/xo-app/backup/new/index.js

```diff
@@ -205,6 +207,10 @@ const destructVmsPattern = pattern =>
         vms: destructPattern(pattern),
       }
 
+const isRetentionLow = (settings, retention) =>
+  retention < RETENTION_LIMIT ||
+  settings.getIn(['', 'fullInterval']) < RETENTION_LIMIT
```

Comment:
Add a comment saying that `fullInterval` can be `undefined` but that the test returns the expected result in that case.